scrapy/utils/benchserver.py

```diff
@@ -16,15 +16,15 @@ def render(self, request):
         total = _getarg(request, 'total', 100, int)
         show = _getarg(request, 'show', 10, int)
```

Comment:
I think ``_getarg`` might also need fixing - elements of ``request.args`` are bytes, and currently the values will always be at default values. At least it was in ``mockserver`` - https://github.com/scrapy/scrapy/pull/1692/files#diff-77ba10fc5f895eef2641f556c8ef6bc1L33